docs/illustrations/FAQs.md

```diff
@@ -0,0 +1,29 @@
+---
+stoplight-id: 26cgp5wk2wbxx
+---
+
+# Illustrations FAQs
+
+## Does the illustration service support drawdown illustrations?
+
+At present, the illustration service only supports accumulation illustrations. Illustrations for drawdown and drawdown transfer will be added in the future. 
+
+## Does the illustration service support fee tiering?
+
+Yes, fee tiers can be accommodate, as per the example in the API documentation.
+
+## Is it possible to add adviser, DFM and product provider charges in one illustration?
+
+Yes, this combination of fees can be added, as per the example in the API documentation. 
```

Comment:
We should probably put some sort of blurb here about where each type of fee should go.
If I remember correctly, adviser fees must go under `adviser`, whilst DFM and platform fees must go under `provider`?